src/main/java/org/jbake/app/ContentStore.java

```diff
@@ -159,17 +162,29 @@ public DocumentList getPublishedPages() {
     }
 
     public DocumentList getPublishedContent(String docType) {
+        return getPublishedContent(docType, false);
+    }
+    
+    public DocumentList getPublishedContent(String docType, boolean applyPaging) {
         String query = "select * from " + docType + " where status='published' order by date desc";
```

Comment:
In future, when paging for multiple content types is supported, LIMIT and SKIP variables can be made docType specific. As with this release, only paging of POST is supported, this fix should do the work!